openmetadata-ui/src/main/resources/ui/src/components/Explore/AdvanceSearchModal.component.tsx

```diff
@@ -28,18 +29,39 @@ export const AdvancedSearchModal: FunctionComponent<Props> = ({
 
   const { t } = useTranslation();
 
+  const handleAdvanceSearchReset = () => {
+    delay(onChangeJsonTree, 100);
+  };
+
   return (
     <Modal
       destroyOnClose
+      closable={false}
+      footer={
+        <Space className="justify-between w-full">
+          <Button
+            className="float-right"
+            size="small"
+            onClick={handleAdvanceSearchReset}>
+            Reset
+          </Button>
+          <div>
```

Comment:
I would recommend using fragments.

Reply:
fragment won't add anything to DOM and I want to add justifyBetween for these two elements. Let me know if Fragment will work here

Reply:
Make sense, fragment will not good fit in that case.